src/wxflow/file_utils.py

```diff
@@ -110,3 +112,29 @@ def _make_dirs(dirlist):
             except Exception as ee:
                 logger.exception(f"Error creating directory {dd}")
                 raise ee
+
+    @staticmethod
+    def _link_files(filelist):
+        """Function to link all files specified in the list
+
+        `filelist` should be in the form:
+        - [target, link name]
+
+        Parameters
+        ----------
+        filelist : list
+                List of lists of [target, link name]
+        """
+        for sublist in filelist:
+            if len(sublist) != 2:
+                raise IndexError(
+                    f"List must be of the form ['target', 'link name'], not {sublist}")
+            target = sublist[0]
+            link_name = sublist[1]
+            if os.path.isdir(link_name):
+                link_name = os.path.join(link_name, os.path.basename(target))
+            logger.info(f"Linking {link_name} to {target}")
+            if not os.path.exists(target):
+                logger.warning(f"WARNING: Target file '{target}' does not exist, will result in dead link!")
+            Path(link_name).symlink_to(target)
+            logger.info(f'Linked {target} to {link_name}')
```

Comment:
Do we really need the log to say:
```
Linking foo to bar
Linked foo to bar
```
Seems like just the second one would be fine?

Reply:
done.